linux/netfilter/netfilter.go

```diff
@@ -14,594 +14,1213 @@ package main
 import "C"
 
 import (
-  "log"
-  "net"
-  "bytes"
-  "errors"
-  "unsafe"
-
-  "C"
-
-  "github.com/google/nftables"
-  "github.com/google/nftables/alignedbuff"
-  "github.com/google/nftables/binaryutil"
-  "github.com/google/nftables/expr"
-  "github.com/google/nftables/xt"
-  linux "golang.org/x/sys/unix"
+	"bytes"
+	"errors"
+	"log"
+	"net"
+	"unsafe"
+
+	"C"
+
+	"github.com/google/nftables"
+	"github.com/google/nftables/alignedbuff"
+	"github.com/google/nftables/binaryutil"
+	"github.com/google/nftables/expr"
+	"github.com/google/nftables/xt"
+	linux "golang.org/x/sys/unix"
+)
+import (
+	"fmt"
 )
 
 type CLogger struct {
-  level C.int
-  function unsafe.Pointer
+	level    C.int
```

Comment:
Can we make sure `go fmt` (https://go.dev/blog/gofmt) is fine with that formatting?